### ISSUE TITLE
Présentation réduite des relations du français

### ISSUE THREAD

**Issue by bguil** [bguil/UD-French-discussion#10]
Dans les deux pages qui présentent les relations du français ([index](http://universaldependencies.org/fr/dep/index.html) et [all](http://universaldependencies.org/fr/dep/all.html)), 
le tableau présente toutes les relations, y compris celles qui sont spécifiques à French-Spoken.
C'est donc difficilement utilisable comme doc pour les autres corpus
Je propose de faire une table réduite seulement avec les relations communes à plusieurs TreeBank et à mettre plus bas dans la page la table complète.
Je pense savoir comment faire ça dans la doc en ligne UD.

La liste des relations étendues communes serait : 

 * `nsubj:pass`
 * `csubj:pass`
 * `aux:pass`
 * `acl:relcl`
 * `flat:name`
 * `flat:foreign`
 * `nmod:poss` (faut-il la garder pour le Français ?)
 * `obl:agent`
 * `aux:caus`
 * à compléter, notamment en fonction des décisions à venir sur les causatifs

**Comment by sylvainkahane** [comment]
ajouter :
acl:cleft
advcl:cleft

et aussi probablement
obl:arg
obl:mod

on ne veut pas nmod:poss pour le français. Ca n’a de sens que pour les langues qui ont un génitif saxon, càd un génitif qui prend la position déterminant.

**Comment by perrier54** [comment]
Je suis d'accord avec la procédure proposée par Sylvain dans un mail que je reprends ici pour la clarté de la discussion : 
_Etape 1: discussion sur le Github du français
Etape 2: transfert de nos décisions au niveau du github général et du guide
Il faut que l’étape 1 se termine par une assignation de tache, càd un responsable transmission dur le github général et un responsable rédaction du guide._
Si on fait le lien avec le tableau proposé par Bruno, cela veut dire que dans le tableau réduit ne figurent que des relations décidées collectivement. Par exemple les relations acl:cleft et advcl:cleft ne doivent pas y figurer pour l'instant. Je pense que cette discipline est nécessaire si nous voulons un guide qui soit utilisable.

**Comment by mcandito** [comment]
Salut,
Je suis d'accord aussi avec cette procédure
(je ne sais pas d'où elle est recopiée, je m'y perds)
